specs/pvs/corla.pvs

```diff
@@ -541,16 +586,27 @@ END ballots_collections
 
 instructions_forms_reports: THEORY
 BEGIN
-  IMPORTING information_systems
+  IMPORTING information_systems, ballot_manifests, election, audits
   
-  report: TYPE
+  report: TYPE = [# name: string #]
 % @review CDOS The contents of this report are underspecified at the
 % moment. See submit_audit_mistake_report below for more information.
-  audit_mistake_report: TYPE FROM report
+  audit_mistake_report: TYPE =
```

Comment:
I'm ok with @nealmcb's recommendation.  I suspect that CDOS has their own terminology.  I had to come up with something since the C.R.S. doesn't actually talk about this in detail.